src/openrct2/ride/Ride.cpp

```diff
@@ -4956,7 +4995,7 @@ static int32_t RideGetTrackLength(const Ride& ride)
     for (const auto& station : ride.GetStations())
     {
         trackStart = station.GetStart();
-        if (trackStart.IsNull())
+        if (trackStart.IsNull() || station.Entrance.IsNull())
```

Comment:
It seems that _all_ changes are neatly put behind `if` checks, with the sole exception of this one. Could you explain it? I’d want to avoid this affecting other modes by accident.